nodejs/week1/circleCalculator.js

```diff
@@ -0,0 +1,23 @@
+// circle calculator class defination
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+  getDiameter() {
+    let diameter = this.radius * 2;
+    console.log(diameter);
+  }
+  getCircumference() {
+    let circumferance = 2 * this.radius * 3.14;
+    console.log(circumferance);
+  }
+
+  getArea() {
+    let area = 3.14 * (this.radius * this.radius);
```

Comment:
You can also get `number` to the power of `x` as `Math.pow(number, x)` or `number**x`. 👍 So `this.radius**2`.